guides/source/routing.md

```diff
@@ -1134,10 +1134,17 @@ For example, here's a small section of the `rake routes` output for a RESTful ro
 edit_user GET    /users/:id/edit(.:format) users#edit
 ```
 
-You may restrict the listing to the routes that map to a particular controller setting the `CONTROLLER` environment variable:
-
+You may restrict the listing to the routes that map to a particular controller using `-c` option or setting the `CONTROLLER` environment variable:
 ```bash
 $ CONTROLLER=users bin/rake routes
+$ bin/rake routes -c users
+$ bin/rake routes -c Users
+$ bin/rake routes -c UsersController
+```
+
+You can grep your routes with `-g`, which filters routes by `name`, `verb`, `path`, or `reqs`.
```

Comment:
What is `reqs` exactly? I assume it means requirements, but a quick Google search for `rails route requirements` found nothing in the current Rails guides — the only match was in 2.3.

I don't think we should mention it.